.github/ISSUE_TEMPLATE/documentation_addition.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation addition
+about: Create a report to help us expand potku-console
+title: "[Documentation update]"
+labels: documentation
+assignees: softgitron, EddieTheCubeHead, joomakelastudent
+
+---
+
+**Describe the supposed addition**
+A clear and concise description of the proposed addition
+
+**Name of the file or section to be added**
```

Comment:
Sometimes there could be a need to update multiple files.

Reply:
Should be fixed in new pr